code/src/AlbumCard.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './albumcard.css'
+import { AlbumImage } from 'AlbumImage'
```

Comment:
You don't need the brackets when you are importing one thing
```suggestion
import AlbumImage from 'AlbumImage'
```